src/main.py

```diff
@@ -38,17 +37,56 @@
 COURSE = os.environ['NOTAS_COURSE_NAME']
 ACCOUNT = os.environ['NOTAS_ACCOUNT']
 
+# Notas repository config
+SHEET_ALUMNOS: str = "Listado"
+COL_EMAIL: str = "E-Mail"
+COL_PADRON: str = "Padrón"
+SHEET_NOTAS: str = "Alumnos - Notas"
+SHEET_DEVOLUCIONES: str = "Devoluciones"
+PREFIJO_RANGO_DEVOLUCIONES: str = "emails"
+RANGO_EMAILS: str = "emailsGrupos"
```

Comment:
Ahi agregue la configuración por parámetro. En este commit dejé los valores hardcodeados. De acá, que valores quieren establecer mediante variables de entorno?

Reply:
Pondria aquellos que vamos a cambiar en caso de estar en "prod" o en caso de estar en "test", si vamos a replicar lo que esta en el spreadsheet de prod al de test, entonces dejemos como esta y no lo pongamos en ninguna env. (A mi me parece mejor esta opción en vez de tener hojas de test en el de prod )

Reply:
Ya que esos valores ahora estan en el `main.py`, te parece si agregamos una variable `STATUS` o `ENV`, y por ejemplo hacemos `SHEET_DEVOLUCIONES = "Devoluciones" if ENV == "PROD" else "Copy of Devoluciones"`?

Reply:
Y aprovecho para preguntar, te parece que haya extraido toda esa configuración al main, preferirias que algunas de estas variables vuelvan al `NotasRepository`, preferirias que sacara tambien `RANGO_NOTAS` que quedo en `NotasRepository` y tener todas esas configuraciones en el `main.py`, a pesar de que no todas serán controladas por variables de entorno?

Reply:
No le veo sentido tener un `Copy of Devoluciones` justamente por lo que comente antes. Si quizas podemos tener una variable que sea `ENV` y consultar si es prod. En caso de ser `test` setamos el spreadsheet de test (Y adaptamos para que el mismo sea igual al de prod, con obviamente datos de prueba). No me gusta la idea de tener hojas de prueba en la spreadsheet de `prod`

Reply:
> Y aprovecho para preguntar, te parece que haya extraido toda esa configuración al main, preferirias que algunas de estas variables vuelvan al NotasRepository, preferirias que sacara tambien RANGO_NOTAS que quedo en NotasRepository y tener todas esas configuraciones en el main.py, a pesar de que no todas serán controladas por variables de entorno?
Pongamosla nuevamente en el repository si no se hacen uso en el main